[error] <input>:3: error: `;` expected but `:` found
[error] abstract class AbstractAsciiImageProvider:
[error]                                          ^
[error] 	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:16)
[error] 	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:16)
[error] 	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
[error] 	at scala.meta.internal.parsers.Reporter.syntaxError(Reporter.scala:17)
[error] 	at scala.meta.internal.parsers.Reporter.syntaxError$(Reporter.scala:17)
[error] 	at scala.meta.internal.parsers.Reporter$$anon$1.syntaxError(Reporter.scala:22)
[error] 	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:429)
[error] 	at scala.meta.internal.parsers.ScalametaParser.syntaxErrorExpected(ScalametaParser.scala:427)
[error] 	at scala.meta.internal.parsers.ScalametaParser.acceptStatSep(ScalametaParser.scala:507)
[error] 	at scala.meta.internal.parsers.ScalametaParser.acceptStatSepOpt(ScalametaParser.scala:510)
[error] 	at scala.meta.internal.parsers.ScalametaParser.statSeqBuf(ScalametaParser.scala:4301)
[error] 	at scala.meta.internal.parsers.ScalametaParser.$anonfun$statSeq$1(ScalametaParser.scala:4290)
[error] 	at scala.meta.internal.parsers.ScalametaParser.$anonfun$statSeq$1$adapted(ScalametaParser.scala:4290)
[error] 	at scala.meta.internal.parsers.ScalametaParser.scala$meta$internal$parsers$ScalametaParser$$listBy(ScalametaParser.scala:560)
[error] 	at scala.meta.internal.parsers.ScalametaParser.statSeq(ScalametaParser.scala:4290)
[error] 	at scala.meta.internal.parsers.ScalametaParser.bracelessPackageStats$1(ScalametaParser.scala:4461)
[error] 	at scala.meta.internal.parsers.ScalametaParser.$anonfun$source$1(ScalametaParser.scala:4464)
[error] 	at scala.meta.internal.parsers.ScalametaParser.atPos(ScalametaParser.scala:328)
[error] 	at scala.meta.internal.parsers.ScalametaParser.autoPos(ScalametaParser.scala:383)
[error] 	at scala.meta.internal.parsers.ScalametaParser.source(ScalametaParser.scala:4429)
[error] 	at scala.meta.internal.parsers.ScalametaParser.entrypointSource(ScalametaParser.scala:4467)
[error] 	at scala.meta.internal.parsers.ScalametaParser.parseSourceImpl(ScalametaParser.scala:121)
[error] 	at scala.meta.internal.parsers.ScalametaParser.$anonfun$parseSource$1(ScalametaParser.scala:118)
[error] 	at scala.meta.internal.parsers.ScalametaParser.parseRuleAfterBOF(ScalametaParser.scala:60)
[error] 	at scala.meta.internal.parsers.ScalametaParser.parseRule(ScalametaParser.scala:55)
[error] 	at scala.meta.internal.parsers.ScalametaParser.parseSource(ScalametaParser.scala:118)
[error] 	at scala.meta.parsers.Parse$.$anonfun$parseSource$1(Parse.scala:30)
[error] 	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:37)
[error] 	at scala.meta.parsers.Api$XtensionParseDialectInput.parse(Api.scala:22)
[error] 	at scala.meta.parsers.Api$XtensionParseInputLike.parse(Api.scala:13)
[error] 	at ScalaCodeAnalyzer$.delayedEndpoint$ScalaCodeAnalyzer$1(ScalaCodeAnalyzer.scala:13)
[error] 	at ScalaCodeAnalyzer$delayedInit$body.apply(ScalaCodeAnalyzer.scala:3)
[error] 	at scala.Function0.apply$mcV$sp(Function0.scala:42)
[error] 	at scala.Function0.apply$mcV$sp$(Function0.scala:42)
[error] 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
[error] 	at scala.App.$anonfun$main$1(App.scala:98)
[error] 	at scala.App.$anonfun$main$1$adapted(App.scala:98)
[error] 	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:619)
[error] 	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:617)
[error] 	at scala.collection.AbstractIterable.foreach(Iterable.scala:935)
[error] 	at scala.App.main(App.scala:98)
[error] 	at scala.App.main$(App.scala:96)
[error] 	at ScalaCodeAnalyzer$.main(ScalaCodeAnalyzer.scala:3)
[error] 	at ScalaCodeAnalyzer.main(ScalaCodeAnalyzer.scala)
[error] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[error] 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[error] 	at sbt.Run.invokeMain(Run.scala:144)
[error] 	at sbt.Run.execute$1(Run.scala:94)
[error] 	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:121)
[error] 	at sbt.Run$.executeSuccess(Run.scala:187)
[error] 	at sbt.Run.runWithLoader(Run.scala:121)
[error] 	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:2044)
[error] 	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1983)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at scala.util.Try$.apply(Try.scala:213)
[error] 	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:369)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[error] 	at java.base/java.lang.Thread.run(Thread.java:1575)
[error] (Compile / run) <input>:3: error: `;` expected but `:` found
[error] abstract class AbstractAsciiImageProvider:
[error]                                          ^
